codegenerator/cli/templates/static/codegen/src/db/TablesStatic.res

```diff
@@ -31,7 +30,7 @@ module EventSyncState = {
       mkField(blockNumberFieldName, Integer, ~fieldSchema=S.int),
       mkField(logIndexFieldName, Integer, ~fieldSchema=S.int),
       mkField(blockTimestampFieldName, Integer, ~fieldSchema=S.int),
-      mkField(isPreRegisteringDynamicContractsFieldName, Boolean, ~fieldSchema=S.bool),
+      mkField(isPreRegisteringDynamicContractsFieldName, Boolean, ~default="false", ~fieldSchema=S.bool),
```

Comment:
Just want to double check that `"false"` is correct and not `false`

Reply:
If needed just leave a comment above

Reply:
It's fine, since we just put it to SQL directly

<img width="727" alt="image" src="https://github.com/user-attachments/assets/577fbc8b-b565-44ac-8cb4-9dc89f057981" />